### PR TITLE
Alerts Pagination Fix and Alerts Download Example Scripts

### PR DESCRIPTION
**Pull request by acviana**
…functionality. For #63

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Documentation
 -------------
 
 A walkthrough of the SDK features and API documentation is available at
-https://python-matchlightsdk.readthedocs.io.
```
Comment (mattgeiger) [inline README.rst]: This should use `https://python-matchlightsdk.readthedocs.io` for the documentation URL. A rebase should resolve this.

File: examples/download-alerts/download-alerts-with-pagination.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    # through the new alerts until we get to this value.
+    last_saved_alert = get_last_saved_alert(db)
+    print(f'Latest alert is number {last_saved_alert.max_alert_number}')
+
+    keep_going = True
+    last_alert_query_parameter = None
+    while keep_going:
+        print(f'Last Alert Query Parameter is {last_alert_query_parameter}')
+        alert_query = ml.alerts.filter(
+            project=project, limit=1, last_alert=last_alert_query_parameter
```
Comment (mattgeiger) [inline examples/download-alerts/download-alerts-with-pagination.py]: I would recomend using a `limit` of more than 1, just to make it clear that they are _pages_, and can get 1 Alert or n Alerts.

File: README.rst (hunk 1 above)
Comment (acviana) [reply README.rst]: Rebased off `development`.

File: examples/download-alerts/download-alerts-with-pagination.py (hunk 2 above)
Comment (acviana) [reply examples/download-alerts/download-alerts-with-pagination.py]: Good point, switched this to a 2 to help illustrate that.

**Pull request merged**